blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/RollupRunnable.java

```diff
@@ -59,13 +62,14 @@
         }
     }
 
-    public RollupRunnable(RollupExecutionContext executionContext, SingleRollupReadContext singleRollupReadContext, RollupBatchWriter rollupBatchWriter) {
+    public RollupRunnable(RollupExecutionContext executionContext, SingleRollupReadContext singleRollupReadContext, RollupBatchWriter rollupBatchWriter, ThreadPoolExecutor enumValidatorExecutor) {
```

Comment:
Do you mean ```HistogramRollupRunnable``` ? I think that's the only subclass of RollupRunnable that I found. Yes I think we can refactor these 2 classes to conform to a common interface. I hate passing around null params in the constructor too.